magento1-module/app/code/local/Divante/VueStorefrontBridge/Helper/Address.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+class Divante_VueStorefrontBridge_Helper_Address extends Mage_Core_Helper_Abstract
+{
+    /**
+     * @param array $streetData
+     *
+     * @return string
+     */
+    public function concatStreetData(array $streetData): string
+    {
+        $concatStreetData = '';
+
+        if (isset($streetData[0], $streetData[1])) {
+            $concatStreetData = $streetData[0] . ' ' . $streetData[1];
+        }
+
+        return $concatStreetData;
+    }
+
+    /**
+     * @param string $streetData
+     *
+     * @return array
+     */
+    public function splitStreetData(string $streetData): array
+    {
+        $streetData = preg_split('/(?=\d)/', $streetData, 2);
```

Comment:
Here I would also suggest working with new lines instead of spaces

Reply:
I think that is a bit harder. 

I have just checked in the database how an Address is saved by magento. 

"1Street Number"
"2Street Number"

And these to are separated by now line. So it is now as easy to move to newline for it.

Reply:
Hm oko, I have to take a look at Magento 1 then. What database table & field did you check?

Reply:
This is the sql which does work for me to get all Addresses. 
`select * from customer_address_entity_text where attribute_id = 25;`